README.md

```diff
@@ -1,3 +1,27 @@
-# resource-exporter
+# resource-exporter
+
 Resource Exportor for volcano scheduling, e.g. NUMA-Aware scheduling.
 
+The DaemonSet is used to obtain the topology information about the CPU resources of the worker node, upload it to ETCD by the [CRD](https://github.com/volcano-sh/apis/tree/master/pkg/apis/nodeinfo/v1alpha1) method for volcano scheduler [numa-aware plugin](https://github.com/volcano-sh/volcano/tree/master/pkg/scheduler/plugins/numaaware)
+## Quick Start Guide
+
+### Compile
+```
+   make image [TAG=XXX]
+```
+
+### Install
+
+#### 1. Edit the file ./installer/numa-topo.yaml

```

Comment:
please describe that the configuration is optional or mandatory.

Reply:
reedit it， described in different parts